### PR TITLE
App strings reworked 11

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: src/resources/i18n/translations/en.json [hunk 1]
```diff
@@ -361,9 +361,9 @@
   "has number": "— one number",
   "has uppercase": "— one big letter",
   "Hide completed episodes by default": "Hide completed episodes",
-  "Hide dividers in lists": "Hide dividers in lists",
+  "Hide dividers in lists": "No dividers in lists",
```
Comment (comradekingu) [inline src/resources/i18n/translations/en.json:364]: Not sure about these

File: src/resources/i18n/translations/en.json [hunk 2]
```diff
@@ -302,8 +302,8 @@
   "Edit My Profile": "Edit Profile",
   "Edit Playlist": "Edit Playlist",
   "Email": "E-mail address",
-  "Enable UnifiedPush notifications": "Enable UnifiedPush notifications",
-  "Enable Lightning Pay switch description": "Experimentally allows adding a Bitcoin wallet to support podcasts through streaming Sats and boosts.",
+  "Enable UnifiedPush notifications": "Allow UnifiedPush notifications",
```
Comment (comradekingu) [inline src/resources/i18n/translations/en.json:305]:
```suggestion
  "Enable UnifiedPush notifications": "UnifiedPush notifications",
```

File: src/resources/i18n/translations/en.json [hunk 3]
```diff
@@ -488,10 +488,10 @@
   "No Wifi Connection": "No Wi-Fi connection",
   "No Wifi Only": "No, Wi-Fi only",
   "Notifications": "Notifications",
-  "Notifications UP help text": "In order to receive push notifications using UnifiedPush, you will need a distributor app installed on your device. After it is installed, then select a distributor from the dropdown above. A popular option for a UP distributor app is called ntfy. Also, you should disable \"battery optimization\" for your distributor app, to make sure your phone is always listening for notifications.",
+  "Notifications UP help text": "Selcet an installed provider (like ntfy) from the dropdown menu to use UnifiedPush. Turn off battery optimization for it to not miss any notifications.",
```
Comment (comradekingu) [inline src/resources/i18n/translations/en.json:491]: Couldn't find this in the menus. Where is it?

Comment (mitchdowney) [reply src/resources/i18n/translations/en.json:491]: The helper text is supposed to be revealed when the "UnifiedPush notifications" toggle is enabled. Maybe it should be showing even when it is not toggled yet though...

**Comment by mitchdowney** [comment]
Thanks @comradekingu these changes look great. If you think we should change that helper text toggle as I mentioned in the other comment please lmk.

**Review by mitchdowney — APPROVED** [review]

File: src/resources/i18n/translations/en.json [hunk 4]
```diff
@@ -658,7 +658,7 @@
   "Start Timer": "Start Timer",
   "Status": "Status:",
   "Stop Timer": "Stop Timer",
-  "Storage data exceeded message": "Local data storage size exceeded. Please unsubscribe from a few podcasts to continue.",
+  "Storage data exceeded message": "Too much local data. Please cancel some subscriptions to continue.",
```
[the remark this answers is not in the document]
Comment (mitchdowney) [reply src/resources/i18n/translations/en.json:661]: @comradekingu no the clips should not affect memory. Our version of "clips" is just a small amount of text data references start and end times within the file. We don't have a "save clip file locally" feature currently.

File: src/resources/i18n/translations/en.json (hunk 1 above)
Comment (mitchdowney) [reply src/resources/i18n/translations/en.json:364]: I think "No" is fine here.

File: src/resources/i18n/translations/en.json (hunk 3 above)
Comment (comradekingu) [reply src/resources/i18n/translations/en.json:491]: Help or no help, I can't find it.
"Enable UnifiedPush notifications" isn't very descriptive.
"Immediate notifications" and then an explanation of how that is done through UP?

Comment (mitchdowney) [reply src/resources/i18n/translations/en.json:491]: @comradekingu oh my bad 🤦‍♂️ you would have to have the F-Droid version running locally. I'm publishing the new version to the F-Droid Store tonight, and it takes about a week before it is downloadable.

In the meantime, if you'd like to download the 4.13.1 beta before that, it can be found at https://files.podverse.fm.

**Comment by mitchdowney** [comment]
@comradekingu we received this feedback about the UP distributor / ntfy explanation. Any thoughts on improving it?

> I would suggest maybe rewording the part about needing ntfy installed. It wasn't obvious at first that I needed to install the ntfy app. Your not provided any options in the drop down if you don't have a compatible app installed. I'm also new to Android/F-Droid so this might just be me.

https://podcastindex.social/@ChadF/110782645318184430

**Comment by comradekingu** [comment]
@mitchdowney 
Seems about right. The idea with
"Set up immediate notifications"
and a more involved explanation below it (whether expandable or not) shown by default would maybe mitigate that.
